tests/tools/onert_run/src/args.cc

```diff
@@ -272,6 +272,7 @@ void Args::Initialize(void)
         "If not given, the model's output sizes are used\n"
         "e.g. '[0, 40, 2, 80]' to set 0th tensor to 40 and 2nd tensor to 80.\n")
     ("num_runs,r", po::value<int>()->default_value(1)->notifier([&](const auto &v) { _num_runs = v; }), "The number of runs")
+    ("fixed_input", "Use same random input data on each run (avaliable on random input)")
```

Comment:
(just for curiosity) Is there a reason why you don't use the same format as others?

Reply:
If we use `po::value<bool>()->default_value(false)->notifier([&](const auto &v) ...`, then we need to set `--fixed_input true` (not `--fixed_input`) to enable this feature.